### DISCUSSION TITLE
/tags endpoint should be available for ApeKey authorization

### DISCUSSION THREAD

**Discussion by fehmer** [monkeytypegame/monkeytype#4421] [Ideas] [answered]
Get results endpoint (https://api.monkeytype.com/documentation/#/results/get_results) includes tags:

```json
"tags": [
                "649c5c18e4860e50da0b762b",
                "649c5c2ae4860e50da0b76c3",
                "649c5c37e4860e50da0b772a",
                "649c5c88e4860e50da0ec5e0"
            ]
```

But there is no way to resolve the tag UIDs to a meaningful text using ApeKey authorization.

**Comment by Miodec** [comment, marked as answer]
Sure thing, added `/users/tags` (will be deployed later today)

**Reply by fehmer** [reply]
works fine, that was fast :) thank you.
